taccsite_cms/settings.py

```diff
@@ -167,14 +168,14 @@ def gettext(s): return s
     "img_file_src": "site_cms/img/favicons/favicon.ico"
 }
 
-INCLUDES_CORE_PORTAL = True
+INCLUDES_CORE_PORTAL = False
```

Comment:
**Uh Oh**: If this is set to `False` here, then all submodules that assume it is true now need to set it to `True`.

Reply:
Oops!! My bad, dev cruft. Thanks for the catch! Will fix and update this.